volatility3/framework/symbols/linux/extensions/elf.py

```diff
@@ -33,15 +36,28 @@ def __init__(
         layer_name = self.vol.layer_name
         symbol_table_name = self.get_symbol_table_name()
         # We read the MAGIC: (0x0 to 0x4) 0x7f 0x45 0x4c 0x46
-        magic = self._context.object(
-            symbol_table_name + constants.BANG + "unsigned long",
-            layer_name=layer_name,
-            offset=object_info.offset,
-        )
+        try:
```

Comment:
Fine with the error handling, but also fine if we just do a length/validity test on the layer?  Either one works.